### ISSUE TITLE
Doesn't account for BitBlt execution time

### ISSUE THREAD

**Issue by laino** [seece/lagmeasure#1]
At least it didn't seem to from a quick reading of the source.

BitBlt isn't a fast operation - depending on the HW it can take a long time to fetch the relevant data from graphics memory and convert to the right pixel format.

Therefore the time to execute BitBlt (pretty much the time to the first frame) should be subtracted when
calculating the final latency.

**Comment by seece** [comment]
Good point. I totally agree. I tried not blitting the data to the external buffer until after the measurements but I think the objects weren't valid anymore afterwards.

Maybe a separate BitBlt benchmark in the beginning could work? Capture e.g. 10 frames and use their average as the estimated latency.

**Comment by TiboQc** [comment]
Came here to say the same. By reducing the size of my window I could reduce the lag average from ~80ms to ~25ms.
Maybe start with a couple of screenshots, measure average time spent in doing this task and reduce it from the estimated lag? Not at all a Windows dev expert.
